docs/guides/index.rst

```diff
@@ -1,11 +1,44 @@
 Guides
 ======
 
-These guides will help walk you through the usage of Read the Docs.
+These guides will help walk you through specific use cases
+related to Read the Docs itself, documentation tools like Sphinx and MkDocs
+and how to write successful documentation.
 
 
+Sphinx & MkDocs how-to guides
```

Comment:
I do feel like we're making the guides harder to find when we add more nesting to the TOC's. I can almost never find the development docs now, since it's nested under 2 levels and really hard to ctrl+f to find. I think we might hit a similar issue here with folks wanting to go to our docs and do `ctrl+f` then search for `PDF`. I don't know the best way to solve this though :/

Reply:
Since the guides were already not on the home page (anymore) I don't know if `ctrl+f` would really help in this instance.

In general, I believe that most users are finding pages by searching google and getting a direct link rather than going to our docs homepage first.

Reply:
Yea, definitely -- but I'd like to support both flows reasonably well. Perhaps promoting search into a landing page-style box in the main content would help here. I need to think more on it.